src/main/webapp/app/exam/manage/suspicious-behavior/suspicious-sessions/suspicious-sessions.component.ts

```diff
@@ -3,7 +3,7 @@ import { SuspiciousExamSessions, SuspiciousSessionReason } from 'app/entities/ex
 import { StudentExam } from 'app/entities/student-exam.model';
 
 @Component({
-    selector: 'jhi-suspicious-sessions',
+    selector: '[jhi-suspicious-sessions]',
```

Comment:
Why?

Reply:
this is an attribute selector instead of a component selector and allows using it e.g., like <tr jhi-suspicious-sessions </tr